docs/guides/nrfconnect_factory_data_configuration.md

```diff
@@ -145,6 +147,15 @@ And replacing existing `sources` with own file names.
 <hr>
 <a name="Generating factory data"></a>
 
+
+## Enabling factory data support
+
+By default, factory data support is disabled in all nRF Connect examples. An nRF Connect device uses pre-defined parameters from Matter Core, which the user should not change. To start using factory data stored in the Flash memory and use the nRF Connect **Factory Data Provider** build an example with following option (replace `nrf52840dk_nrf52840` with your board name):
```

Comment:
As nRF Connect device I mean the device that uses our SoCs, and works based on nRF Connect platform code. 

According to pre-defined parameters, in the Matter repository there are some .h and .cpp files containing all mocked factory data parameters that are used instead of real factory data. I think user shouldn't edit Matter core so I guess these parameters shouldn't be changed. Our factory data parameters are saved in kConfigs, that users can change. But users may want to create factory data in own way, and we decided to agree with that, but both default parameters and our implementation must be disabled. But maybe, you're right, maybe I used wrong words here and there are some misunderstandings.